proposals/Exceptions.md

```diff
@@ -240,19 +234,21 @@ If the `kind` is `Exception`:
 
 | Field | Type | Description |
 |-------|------|-------------|
-| type  | `varuint32` | exception index to table in the *exception type section* |
+| tag   | varuint32 | index into exception section |
```

Comment:
Clearly, we want exceptions to be imported, so that one module can throw it to another?

Separate entries in the exception section can have the same data fields, and still define different exceptions. For example, how could you tell if one is a C++ exception, and the other is not?